### PR TITLE
new package: nlopt

### LINKED ISSUE #2647: Package request: nlopt
#143

### PR DESCRIPTION
**Pull request by xtkoba**
Closes #2647.

### CODE REVIEW

File: packages/nlopt/build.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://nlopt.readthedocs.io/
+TERMUX_PKG_DESCRIPTION="Library for nonlinear optimization"
+TERMUX_PKG_LICENSE="LGPL-2.1, MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.6.2
```
Comment (Grimler91) [inline packages/nlopt/build.sh:5]: Any particular reason for not packaging the latest release (2.7.0)?

Comment (xtkoba) [reply packages/nlopt/build.sh:5]: Ah, I did not notice that a newer version had been released just the day before.

**Comment by xtkoba** [comment]
I might have broken something...

**Comment by xtkoba** [comment]
I don't know how to fix this. Discarded.

**Pull request closed without merging**